lib/extensions/active_record_extension.rb

```diff
@@ -4,8 +4,7 @@ def swap_error_key(from, to)
   end
 
   def destroy_on_blank(attributes, *keys)
-    attributes.each do |attribute|
-      item = attribute[1]
+    attributes.each do |k, item|
```

Comment:
Because the `k` parameter is unused, it should be prefixed with an underscore:
```ruby
attributes.each do |_k, item|
```
https://github.com/rubocop-hq/ruby-style-guide#underscore-unused-vars

Reply:
I'll take that one change and give you 2 changes! :)